ansible/playbook.yaml

```diff
@@ -0,0 +1,237 @@
+        - name: KMM - Delete the KMM operator controller pod for `ConfigMap` changes to take effect.
+          shell: oc get pods -n openshift-kmm | grep -i "kmm-operator-controller-" | awk '{print $1}' | xargs oc delete pod -n openshift-kmm
+        - name: KMM - wait 10 seconds until KMM operator is up and running
+          pause:
+            seconds: 10
+      - name: IDPO - Install Intel Device Plugins Operator
+        tags: 
+          - idpo
+        block:
+          - name: IDPO - Install Intel Device Plugins Operator
+            k8s:
+              state: present
+              src: "../device_plugins/install_operator.yaml"
+              wait: yes
+          - name: IDPO - Wait until the inteldeviceplugins-controller-manager Deployment is available
+            k8s_info:
+              kind: Deployment
+              wait: yes
+              name: inteldeviceplugins-operator-controller
+              namespace: openshift-operators
+              wait_condition:
+                type: Available
+                status: 'True'
+                reason: MinimumReplicasAvailable
+    - name: NFD - Install NFD CRs
+      block:
+        - name: NFD - Create NFD discovery CR
+          k8s:
+            state: present
+            src: "../nfd/node-feature-discovery-openshift.yaml"
+            wait: yes 
+        - name: NFD - Create NFD rules instance CR
+          k8s:
+            state: present
+            src: "../nfd/node-feature-rules-openshift.yaml"
+            wait: yes
+    - name: KMM - Deploy Pre-built Out-of-Tree Intel Data Center GPU Driver Container for OpenShift using KMM
+      block: 
+        - name: KMM - Install KMM pre-build Module CR
+          k8s:
+            state: present
+            src: "../kmmo/intel-dgpu.yaml"
+            wait: yes
+        - name: KMM - Wait 30 secs to load OOT driver container and the KMM label is set to ready
+          pause:
+            seconds: 30
+        - name: KMM - Wait for node icx-flex140-3 with kmm label ready
+          community.kubernetes.k8s_info:
+            api: v1
+            kind: Node
+            label_selectors:
+              - "kubernetes.io/hostname = icx-flex140-3"
+          register: cluster_nodes
+          until:
+            - cluster_nodes.resources is defined
+          retries: 90
+          delay: 10
+        - name: KMM - KMM Set Ready label true. Fail if false
+          set_fact:
+            is_kmm_label_ready: "{{ 'kmm.node.kubernetes.io/openshift-kmm.intel-dgpu.ready' in cluster_nodes.resources[0].metadata.labels }}"
+          failed_when: is_kmm_label_ready == false
+    - name: IDPO - Install GPU plugin
+      block: 
+        - name: IDPO - Create Intel GPU device plugin
+          k8s:
+            state: present
+            src: "../device_plugins/gpu_device_plugin.yaml"
+            wait: yes
+        - name: IDPO - Wait 30 seconds for GPU device plugin to be ready
+          pause:
```

Comment:
It is possible to use condition wait? :-)

Reply:
Done. Wait not needed here. :)